README.rst

```diff
@@ -674,9 +676,13 @@ dependencies with ``select`` expressions (Gazelle does this automatically).
       }),
   )
 
-To build a specific `go_binary`_ or `go_test`_ target for a target platform,
-set the ``goos`` and ``goarch`` attributes on that rule. This is useful for
-producing multiple binaries for different platforms in a single build.
+To build a specific `go_binary`_ target for a target platform, use the
```

Comment:
Could you mention here that one can also specify a different SDK version?

Reply:
Done!